tests/search/search_tests.py

```diff
@@ -169,6 +176,52 @@ def test_search_terms(self):
         request.user = self.tester
         response = search_terms(request)
         result = {}
+        try:
+            result = json.loads(response.content)
+        except json.decoder.JSONDecodeError:
+            print("Failed to parse search result")
+        self.assertTrue("terms" in result and len(result["terms"]) == 0)
+        remove_perm("no_access_to_nodegroup", self.tester, nodegroup)
+        new_tile.delete()
+        resource.delete()
+        self.sync_es(se)
+
+    def test_search_terms_permitted_user(self):
+        """
+        Test search terms method with a permitted user
+
+        """
+        admin_user = User.objects.get(username="admin")
+        nodeid = "c9b37b7c-17b3-11eb-a708-acde48001122"
+        tileid = "bebffbea-daf6-414e-80c2-530ec88d2705"
+        resourceinstanceid = "745f5e4a-d645-4c50-bafc-c677ea95f060"
+        resource = Resource(uuid.UUID(resourceinstanceid))
+        resource.graph_id = "c9b37a14-17b3-11eb-a708-acde48001122"
+
+        resource.save(user=admin_user, transaction_id=uuid.uuid4())
+        tile_data = {}
+        tile_data[nodeid] = {
+            "en": {"value": "Alpha Gamma Delta Eta Epsilon", "direction": "ltr"}
+        }
+        new_tile = Tile(
+            tileid=uuid.UUID(tileid),
+            resourceinstance_id=resourceinstanceid,
```

Comment:
Thanks for using a UUID for tile -- let's do that for resource and nodegroup also, so we don't add tech debt on top of #10877 (I'm hoping we'll get a CI baseline for that and "plug the leak before filling the leak")